redi-connect-front/src/components/molecules/Title.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Container, Section, Content } from 'react-bulma-components'
+import './Title.scss'
+
+interface Props {
+  teaser: string
+  colorCode?: string
+  headline: string
+}
+
+const Title = ({ teaser, colorCode = '##ea5b29', headline }: Props) => {
+  return (
+    <Container>
+      <Section className="custom-title__wrapper">
+        <Content className="custom-title__teaser">{teaser}</Content>
+        <div className="custom-title__line" style={{ borderColor: colorCode }} />
```

Comment:
I usually try to separate styling from the code. My first pick to solve this would be to add a pseudo element with the line. Have you checked out the heading on the login page? 
We can keep this as you build for now, this is also working, I'm just saying, in case you would like to check out another solution you can visit the version I built.

Reply:
The idea here is to re-use this Title component because it comes up pretty often. I think the color for the seperator is almost always orange but in case it's not we can change it with a prop. I got rid of it ... we can change it back if a different color comes up.

Reply:
Ok, I'm fine with the idea basically. 
Just keep in mind, that you always optimise the things minimal, especially in the early faze like this. It would be enough that when the next time you need to develop a heading like this then you create the component. Also, is there a real indication that the color could be different? I usually develop for the case I am doing atm, this way there is no need to overthink. Saves me time and I will produce nicer code.